### PR TITLE
Feature/battle fight

### PR DESCRIPTION
**Pull request by Sota-hub**

### CODE REVIEW

File: src/components/BattleScreen/BattleDisplay/BattleDisplay.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        <p>{enemy.firstEnemy.forms[0].name}</p>
+        <progress
+          className={classes.hp_bar}
+          id="hp"
+          value="100"
+          max="100"
+        ></progress>
+      </div>
+      <div className={classes.user_image}>
+        <img
```
Comment (eddybanks) [inline src/components/BattleScreen/BattleDisplay/BattleDisplay.js:20]: Make sure you add alt values for each img.

File: src/components/BattleScreen/BattleFight/BattleFight.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import classes from "./BattleFight.module.scss";
+
+const BattleFight = () => {
+  return (
+    <div className={classes.menu_details}>
+      <a href="#" className={classes.button}>
```
Comment (eddybanks) [inline src/components/BattleScreen/BattleFight/BattleFight.js:6]: We might have to work on the hrefs before merging to production

**Review by eddybanks — COMMENTED** [review]
Please check the netlify logs for errors and fix them in the next commit. Most of the warnings/errors have to do with <a href> tags and

File: src/components/BattleScreen/BattleDisplay/BattleDisplay.js (hunk 1 above)
Comment (Sota-hub) [reply src/components/BattleScreen/BattleDisplay/BattleDisplay.js:20]: Thanks, dealt with it

**Pull request closed without merging**